src/components/stepper/stepper.stories.ts

```diff
@@ -191,3 +191,21 @@ export const arabicNumberingSystem_TestOnly = (): string => html` <calcite-stepp
     </calcite-notice>
   </calcite-stepper-item>
 </calcite-stepper>`;
+
+export const verticalLayout_TestOnly = (): string => html`<calcite-stepper layout="vertical" scale="s">
```

Comment:
@benelan WDYT about adding a screenshot test convention for covering scale-specific issues in a single story? @Elijbet You've added similar stories, right?

Reply:
A convention would be good. Here are some examples. Sometimes for the layout testing, I include other prop variations to see if they affect/interact with the prop I'm testing at different scales(eg props: heading/description/long title/slots, etc).

```mediumIconForLargeAccordionItem_TestOnly
withActions
actionsEndNoQueue_TestOnly
actionsEndQueued_TestOnly
mediumIconForLargeComoboboxItem_TestOnly
mediumIconForLargeDropdownItem_TestOnly
mediumIconForLargeScaleStyling_TestOnly
largeScaleLayout_TestOnly

Combining non default scale test with other tests:

hebrewNumberingSystemAndMediumIconForLargeInputStyling_TestOnly
disabled_TestOnly
laoNumberingSystemAndMediumIconForLargeInput_TestOnly
darkModeFrenchLocaleAndLargeScaleGetsMediumChevron_TestOnly
disabledAndLargeScaleGetsMediumChevron_TestOnly
disabledTabsAndMediumIconsForLargeTabsTitle_TestOnly```

Reply:
if the UI isnt affected by other props apart from `scale`, i think its better to include all the scales in a single test so that we can saveup on screenshots. If we had to include other props and see how it behaves , then we may have tests them separately as mentioned above. Thoughts @Elijbet , @jcfranco , @benelan ?

Reply:
Not sure, here for example I put all variations in the same screenshot https://github.com/Esri/calcite-components/pull/6403
```export const iconStartAndActionsEnd = (): string => html`
  <div style="width: 650px">
    <calcite-tree style="margin: 80px" scale="l"> ${iconStartLargeActionAndActionsEnd} </calcite-tree>
    <calcite-tree style="margin: 80px" scale="m"> ${slottedSmallAction} </calcite-tree>
    <calcite-tree style="margin: 80px" scale="s"> ${slottedSmallActionsEndAndAction} </calcite-tree>
  </div>
`;```

Reply:
makes sense. The same applies for this PR too, me thinks.

Reply:
Yeah I like that idea of putting them all in the same story. I had been recommending setting different scales for existing stories, as long as it doesn't conflict with the existing coverage. For example changing the scale to `s` on a story testing dark mode or a different lang/numberingSystem. We would just need to alter the story name to mention both coverage cases.

If there isn't a way to alter an existing story, I'd say putting all three in the same story is our second best option. 

In terms of adding conventions, I'd like to wait another month or two when we'll know what our Chromatic snapshot usage is under normal conditions. The first month we went over the limit due to testing during initial setup. The second month we were way under the limit due to holidays. So I'd like to see if we have any wiggle room before deciding how we should handle the situation in the long term.

And to clarify, we should definitely be adding coverage to scale issues as we fix them right now, but if we have some wiggle room it could be nice to add scale coverage to all of our components to catch issues before they arise. Just not sure the best way to do that until we know our usage.